rampwf/utils/testing.py

```diff
@@ -37,18 +37,31 @@ def assert_title(ramp_kit_dir='.'):
     print_title('Testing {}'.format(problem.problem_title))
 
 
-def assert_data(ramp_kit_dir='.', ramp_data_dir='.'):
+def assert_data(ramp_kit_dir='.', ramp_data_dir='.', data_label=None):
     problem = assert_read_problem(ramp_kit_dir)
-    print_title('Reading train and test files from {}/data ...'.format(
-        ramp_data_dir))
-    X_train, y_train = problem.get_train_data(path=ramp_data_dir)
-    X_test, y_test = problem.get_test_data(path=ramp_data_dir)
+    if data_label is None:
```

Comment:
I would be more natural to branch inside get_train_data and get_test_data

I am sure you don't do think to stay backward compatible but maybe can you add a comment to explain this?

Reply:
Indeed, I simplified that this code a bit.